### PR TITLE
applications/misc/yate: Bump from version 2 to 5.4.x

### PR DESCRIPTION
**Pull request by victoredwardocallaghan**
Signed-off-by: Edward O'Callaghan eocallaghan@alterapraxis.com

### CODE REVIEW

File: pkgs/applications/misc/yate/default.nix [hunk 1]
```diff
@@ -28,11 +28,13 @@ composableDerivation.composableDerivation {} ( fixed : {
...
-  meta = { 
+  meta = {
     description = "YATE - Yet Another Telephony Engine";
     homepage = http://yate.null.ro/;
-    license = ["GPL" "MPL"]; # Yate's license is GPL with an exception for linking with OpenH323 and PWlib (licensed under MPL).
-    maintainers = [ lib.maintainers.marcweber ];
+    # Yate's license is GPL with an exception for linking with
+    # OpenH323 and PWlib (licensed under MPL).
+    license = ["GPL" "MPL"];
+    maintainers = [ lib.maintainers.funfunctor ];
```
Comment (lethalman) [inline pkgs/applications/misc/yate/default.nix]: You shouldn't remove maintainers like this. cc @MarcWeber

Comment (MarcWeber) [inline pkgs/applications/misc/yate/default.nix]: I don't use Yate ATM, so without being asked it happens that I'm fine.

**Comment by lethalman** [comment]
I didn't even built this locally, given that you didn't change the sha of the package.

**Comment by pSub** [comment]
I have updated yate in 94d177fc5ec7e73e8af947d2a14d4ff08d675a58.

**Pull request closed without merging**